bin/morgue.js

```diff
@@ -5254,17 +5282,26 @@ function objectPrint(g, object, renderer, fields, runtime) {
 
         if (a === 'callstack')
           continue;
-
-        console.log('  ' + a.yellow.bold + ': ' + fieldFormat(ob[a], fields[a]));
+        const prettyValue = fieldFormat(ob[a], fields[a]);
```

Comment:
I would think that we would want to be faithful to the original when exporting CSVs for import into other systems, not format it through Morgue's stdout logic.  This is lossy, and also adds in units which means that you can't i.e. just parse an int column as ints when trying to import into (say) Postgres and that things like BigQuery's automatic type detection will have problems. Looking down the list of the rules in `fieldFormat`, we need about half of them and need to drop the other half; I suggest a separate function near `fieldFormat`, `fieldFormatCsv`.

Reply:
Agree. Changing.